addons/account/wizard/account_move_send_views.xml

```diff
@@ -11,6 +11,14 @@
                 <field name="company_id" invisible="1"/>
                 <field name="move_ids" invisible="1"/>
                 <field name="mode" invisible="1"/>
+                <field name="show_pdf_template_menu" invisible="1"/>
+                <group>
+                    <field
+                        name="pdf_template_id"
+                        invisible="not show_pdf_template_menu"
+                        class="w-50"
+                        options="{'no_create': True, 'no_edit': True}"/>
+                </group>
```

Comment:
The warnings will be below that, is it what we want ? wouldn't it be better to put that in the options div or just above it ?